### ISSUE TITLE
Project stat percentages

### ISSUE THREAD

**Issue by Jazza-231** [ScratchAddons/ScratchAddons#7241]
### Why this would be helpful

To see the relative amount of likes, favs, and remixes compared to the number of views.

### How this addon works

Hover over the counts to see the percentage

### Possible alternatives

N/A

### Additional context

I have made this, wl just suggested to make an issue, see #percent

**Comment by Jazza-231** [comment]
(to be clear, this is fully ready)

**Comment by mybearworld** [comment]
This makes Scratch seem like a popularity contest, in my opinion. SA even has an addon to remove these kinds of stats.

**Comment by BlueGoat11** [comment]
I agree, think we should just stick with the hearts, likes, and views.On Mar 3, 2024, at 9:34 AM, mybearworld ***@***.***> wrote:﻿
This makes Scratch seem like a popularity contest, in my opinion. SA even has an addon to remove these kinds of stats.

**Comment by Jazza-231** [comment]
> This makes Scratch seem like a popularity contest, in my opinion. SA even has an addon to remove these kinds of stats.

I think of it as a simple metric of how well received the project is... Let's be clear, scratch is social media. Stats exist already, all this does is a simple calculation for you, which you can do easily yourself

**Comment by WorldLanguages** [comment]
I guess this is fine... But we have too many addons already, I don't know what to think about adding new addons like this.

**Comment by Samq64** [comment]
I think it might have already been mentioned on the PR but this could be merged with the lat edited tooltip add-on which is also tiny right now.

**Comment by Jazza-231** [comment]
> I think it might have already been mentioned on the PR but this could be merged with the lat edited tooltip add-on which is also tiny right now.

If someone reminds me in about 10 hours I'll do that, and I'll rename it and make that addon use this nice popup (in fact, should I make this a kind of library or add it to the addon API??)

**Comment by Samq64** [comment]
I think normal tooltips are fine unless you find a way to make yours work with touch.

**Comment by Jazza-231** [comment]
> I think normal tooltips are fine unless you find a way to make yours work with touch.

Uhmm I find it highly unlikely that people will use touch but I'll look into it

**Comment by mybearworld** [comment]
> Uhmm I find it highly unlikely that people will use touch but I'll look into it

See #6789

**Comment by WorldLanguages** [comment]
> I think it might have already been mentioned on the PR but this could be merged with the lat edited tooltip add-on which is also tiny right now.

Hmm IMO they don't really fit into each other that well...
